server/controllers/request.js

```diff
@@ -1,13 +1,24 @@
 const Request = require("../models/Request");
 const asyncHandler = require("express-async-handler");
 
-// @route GET /requests
+// @route GET /requests/sitter
 // @desc get requests for logged in user
 // @access Private
 exports.getRequests = asyncHandler(async (req, res) => {
+  const { isSitter } = req.query ?? "false";
```

Comment:
Let's consider adding a field to the Profile model to distinguish the type of user instead :) 

This issue is for the same purpose: https://github.com/hatchways/team-bamboo/issues/50 (Check with Simon, in case he's done you can merge that branch into this one or work on the issue if it's okay with him) 😄